pom.xml

```diff
@@ -96,7 +96,8 @@
         <maven.plugin.build.helper.version>3.2.0</maven.plugin.build.helper.version>
         <maven.plugin.compiler.version>3.8.1</maven.plugin.compiler.version>
         <!-- DO NOT bump 4.4.0, see https://github.com/NetEase/kyuubi/pull/441 -->
-        <maven.plugin.scala.version>4.3.0</maven.plugin.scala.version>
+        <!-- DO NOT bump 4.5.3, see https://github.com/NetEase/kyuubi/issues/708 -->
+        <maven.plugin.scala.version>4.5.3</maven.plugin.scala.version>
```

Comment:
revert it to `4.3.0`

Reply:
@wuchunfu

Reply:
@yaooqinn I has now been revert and add a new comment here and skip this version too.